engine/runtime/src/main/scala/org/enso/compiler/core/IR.scala

```diff
@@ -2913,6 +2913,84 @@ object IR {
       /** @inheritdoc */
       override def showCode(indent: Int): String = name
     }
+
+    /** A representation of the name `self`, used to refer to the current type.
```

Comment:
```suggestion
    /** A representation of the name `Self`, used to refer to the current type.
```
Shouldn't this be capitalised?